clusterloader2/pkg/measurement/common/ooms_tracker.go

```diff
@@ -42,7 +42,7 @@ const (
 )
 
 var (
-	oomEventMsgRegex = regexp.MustCompile(`Kill process (\d+) \((.+)\) score \d+ or sacrifice child\nKilled process \d+ .+ total-vm:(\d+kB), anon-rss:\d+kB, file-rss:\d+kB.*`)
+	oomEventMsgRegex = regexp.MustCompile(`Killed process (\d+) \((.+)\) total-vm:(\d+kB), anon-rss:\d+kB, file-rss:\d+kB.*`)
```

Comment:
Shouldn't we try to have both?

Reply:
I'm not sure I follow but here's the context: https://github.com/kubernetes/node-problem-detector/issues/480#issuecomment-714467335.

Basically, I'm adapting the regex to handle OOM events emitted by both the old and the new NPD here.

Reply:
If you are replacing `Kill` with `Killed` it would not work with old NPD.

Reply:
It will - I'm simply removing the prefix of the message we observe in the NPD's kernel monitor.

Reply:
You are right, I did not notice the second part.

Reply:
Could you add some unit tests? It could be in another commit so it would be easier to backport.

Reply:
Sure, I will open a separate PR for that.

Reply:
As backports already have merged, ok.